src/core/query.ts

```diff
@@ -106,6 +106,7 @@ interface ContinueAction {
 interface SetStateAction<TData, TError> {
   type: 'setState'
   state: QueryState<TData, TError>
+  setStateOptions?: SetStateOptions
```

Comment:
Think this change is not needed anymore?

Reply:
Done.